challenges/revenue.js

```diff
@@ -0,0 +1,20 @@
+function revenue(transactions){
+
+  let obj = {};
```

Comment:
Your solution is fantastic, but just wanted to point out that you could alternatively use "reduce" rather than declaring an empty object and pushing stuff into it. You would reduce the array of transactions, with the accumulator being an {}. Reduce can be super confusing, so just wanted to point out this out as somewhere you could use it!